gsrs-module-substance-example/src/test/resources/application-test.conf

```diff
@@ -4,39 +4,34 @@ spring.main.allow-bean-definition-overriding=true
 
 ix.gsrs.protein.mw.percent.tolerance=1.0
 substance.renderer.configPath=substances-default-renderer.json
-logging.level.gsrs.module.substance.scrubbers.basic=trace
-logging.level.example.exports.expanders=trace
-logging.level.gsrs.module.substance.expanders.basic=trace
-logging.level.gsrs.module.substance.repository=trace
 
 approval_id_code_system="UNII from FDA"
 logging.level.root=error
-
-#logging.level.example.substance.processor=trace
-#logging.level.gsrs.module.substance.processors=trace
-#logging.level.ix.ginas.utils.validation.validators=trace
+logging.level.example.imports=trace
+logging.level.gsrs.imports=trace
+logging.level.example.substance.processor=trace
+logging.level.gsrs.module.substance.processors=trace
+logging.level.ix.ginas.utils.validation.validators=trace
 logging.level.ix.ginas.utils.validation=trace
-#logging.level.gsrs.module.substance.indexers.SubstanceDefinitionalHashIndexer=trace
-#logging.level.example.substance.crud=trace
-#logging.level.example.substance.datasearch=trace
-
-#logging.FileHandler.pattern="d:/temp/gsrs/gsrs3.tests.log"
-#java.util.logging.FileHandler.limit=50000
-#java.util.logging.FileHandler.count=1
-#java.util.logging.FileHandler.formatter=java.util.logging.SimpleFormatter
-#java.util.logging.FileHandler.level=FINE
-#java.util.logging.FileHandler.level=SEVERE
-logging.level.gsrs=ERROR
+logging.level.gsrs.module.substance.indexers.SubstanceDefinitionalHashIndexer=trace
+logging.level.example.substance.crud=trace
+logging.level.example.substance.datasearch=trace
+logging.level.example.substance.validation=trace
+logging.level.example.imports=trace
+logging.level.gsrs.module.substance.importers=trace
+logging.level.gsrs.data_exchange.model=trace
+logging.level.gsrs.controller=trace
+logging.level.example.imports.readers=trace
+logging.level.gsrs.dataexchange.processing_actions=trace
+
+
+logging.level.gsrs=TRACE
 logging.level.example.substance=ERROR
 logging.level.ix.ginas=ERROR
 logging.level.org.springframework=ERROR
+logging.level.example.imports=trace
```

Comment:
A lot of these trace logs may need to change for jenkins. Too verbose a log makes the build fail due to the log getting too big. Can we comment out these?